kujira/app/services/enevt-handler-service.js

```diff
@@ -0,0 +1,33 @@
+
+import Ember from 'ember';
+var run= Ember.run;
+var socket;
+var hostName ='';
```

Comment:
Później zrobię commita
On Apr 2, 2016 8:49 PM, "Przemysław Niekraś" <notifications@github.com>
wrote:

> In kujira/app/services/enevt-handler-service.js
> <https://github.com/KujiraProject/kujira-web/pull/4#discussion_r58297915>:
>
> > @@ -0,0 +1,33 @@
> > +
> > +import Ember from 'ember';
> > +var run= Ember.run;
> > +var socket;
> > +var hostName ='';
>
> Value should be taken from config/environment.js: ENV.APP.API_HOST and
> used in io.connect directrly.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/KujiraProject/kujira-web/pull/4/files/0940445c6fef7e7cef1987ec1aa69535aea47cb9#r58297915>
>